VATSpy.dat

```diff
@@ -18326,8 +18326,10 @@ ZMUB|Ulaan Baatar|ZMUB|
 [UIRs]
 ADR_U|Adria Radar Upper|LJLA,LDZO,LQSB,LYBA,LWSS,LAAA
 AFRN|North Africa Control|DAAA,DRRR,DTTC,FTTT,GMMM,HLLL
+AFRC|Africa Central Control|FLFI,FNAN,FSSS,FWLL,FZZA,HKNA,HTDC,HUEC
 AFRE|East Africa Control|HAAA,HCSM,HHAA,HSSS
-AFRS|Africa South Control|FACA,FAJA,FBGR,FIMM,FLFI,FMMM,FNAN,FQBE,FVHF,FWLL,FYWH,HKNA,HTDC,HUEC
+AFRS|Africa South Control|FACA,FAJA,FBGR,FIMM,FMMM,FQBE,FVHF,FYWH
+AFRW|Africa West Control|DGAC,DIII,DNKK,FCCC,GLRB,GOOO,GVSC

```

Comment:
Just a notice: This will not make GOOO OCA light up since its defined as a OCA position.

Reply:
Agreed but if we can do something about this in the future it would be useful.